internal/auth0/branding/resource.go

```diff
@@ -140,9 +140,10 @@ func readBranding(ctx context.Context, d *schema.ResourceData, m interface{}) di
 func updateBranding(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	api := m.(*management.Management)
 
-	branding := expandBranding(d.GetRawConfig())
-	if err := api.Branding.Update(branding); err != nil {
-		return diag.FromErr(err)
+	if branding := expandBranding(d.GetRawConfig()); branding.String() != "{}" {
```

Comment:
Just curious, is `!= "{}"` the only way to check for resource emptyness?

Reply:
We could as well check within the expandBranding if the branding is `== (management.Branding{})`. However checking if the resulting string is `{}` is a bit easier IMO, less code to add.

Reply:
IT would be nice to have an `IsEmpty` func within the Go SDK tho.

Reply:
> However checking if the resulting string is {} is a bit easier IMO, less code to add.

Could the string representation change at some point?

Reply:
Unlikely as the string representation is the JSON representation.